data/human/human missions.txt

```diff
@@ -6890,3 +6890,89 @@ mission "Hogshead's Dilemma II"
 			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
 			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
 				decline
+
+
+
+mission "Lost Dog"
+	name "Bring a dog back to <planet>"
```

Comment:
```suggestion
	name "Bring a dog back that've been lost to <destination>."
```

Reply:
The field for the name in the mission panel is quite small, so we have to be careful about how much we put in there.